plugins/rxp/io/RxpPointcloud.cpp

```diff
@@ -52,24 +52,28 @@ Dimension::Id getTimeDimensionId(bool syncToPps)
 }
 
 
+Point::Point(scanlib::target target, unsigned int returnNumber, unsigned int numberOfReturns)
+    : target(target)
+    , returnNumber(returnNumber)
+    , numberOfReturns(numberOfReturns)
+{}
+
+
 RxpPointcloud::RxpPointcloud(
         const std::string& uri,
         bool syncToPps,
-        bool minimal,
         bool reflectanceAsIntensity,
         float minReflectance,
         float maxReflectance,
         PointTableRef table)
     : scanlib::pointcloud(syncToPps)
-    , m_view(new PointView(table))
-    , m_idx(0)
     , m_syncToPps(syncToPps)
-    , m_minimal(minimal)
     , m_reflectanceAsIntensity(reflectanceAsIntensity)
     , m_minReflectance(minReflectance)
     , m_maxReflectance(maxReflectance)
     , m_rc(scanlib::basic_rconnection::create(uri))
     , m_dec(m_rc)
+    , m_points()
```

Comment:
Unnecessary.

Reply:
Habit carry-over from "you must initialize everything" languages. Same as above comment, happy to remove if you'd like, but recommend putting something in the style guide about it to help prevent future churn?